### PR TITLE
Add Cirrus CI, remove unused content, add btests for `update-changes`

### PR DESCRIPTION
**Pull request by ckreibich**
I realized that I'm really uneasy changing `update-changes` (over in #33, which I'll revisit once this has landed) and since we didn't have tests for it, I figured I'd add some as part of removing old Travis setups.

### CODE REVIEW

**Comment by ckreibich** [comment]
Turns out there's a bit more cruft in that repo, so I'm removing more things. :-)

File: zeek-cut/CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,3 @@ set(zeekcut_SRCS
 add_executable(zeek-cut ${zeekcut_SRCS})
 
 install(FILES zeek-cut.1 DESTINATION ${ZEEK_MAN_INSTALL_PATH}/man1)
-
-# Install wrapper script for Bro-to-Zeek renaming.
-include(InstallSymlink)
-InstallSymlink("${CMAKE_INSTALL_PREFIX}/bin/zeek-wrapper" "${CMAKE_INSTALL_PREFIX}/bin/bro-cut")
```
Comment (timwoj) [inline zeek-cut/CMakeLists.txt:11]: Looks like this was taken care of by https://github.com/zeek/zeek-aux/commit/67f10f8e010f59b0913f289d1160657c70ccf0ed, if you want to rebase.

File: testing/Makefile [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-DIAG=diag.log
 BTEST=../../btest/btest
+DIAG=diag.log
 
 all: cleanup btest-verbose
 
 # Showing all tests.
 btest-verbose:
-	@$(BTEST) -j -f $(DIAG)
+	@test -f $(BTEST) && bt=$(BTEST) || bt=btest; $$bt -j -f $(DIAG)
```
Comment (timwoj) [inline testing/Makefile:8]: Would `-x` be better than `-f` here? That tests that it not only exists, but is also executable.

Comment (timwoj) [reply testing/Makefile:8]: Also, out of curiosity, why the double `$$` in `$$bt`?

**Review by timwoj — CHANGES_REQUESTED** [review]
Just some minor nits and questions.

File: testing/Makefile (hunk 2 above)
Comment (bbannier) [reply testing/Makefile:8]: [Escaping for `$`](https://www.gnu.org/software/make/manual/html_node/Rule-Syntax.html#index-rules_002c-and-_0024) so `bt` is evaluate in the shell session 🦄 .

Comment (ckreibich) [reply testing/Makefile:8]: Good point re `-x`, will do Tim. Benjamin's on it re `$$`. I don't think this is super elegent btw — I initially tried to express this properly in the Makefile but it's a pain to make portable, so settled for this.

**Review by timwoj — APPROVED** [review]

**Pull request merged**